tests/test.py

```diff
@@ -3511,6 +3511,47 @@ def create_key_store_with_client_id_from_cert(self, extractor, dir_name):
             timeout=PROCESS_CALL_TIMEOUT)
 
 
+class TestAcraKeysWithRedis(unittest.TestCase):
+    def setUp(self):
+        self.master_key = get_master_key()
+        self.client_id = 'keypair1'
+
+    def test_read_command_keystore(self):
+        with tempfile.TemporaryDirectory() as keys_folder:
```

Comment:
as I understand, we don't need FS folder for redis storage. we just pass some folder path as parameter that will be used as key in redis. so we don't need to create any real directory

Reply:
If I am not mistaken actually it is used for creating path to keys inside Redis, i suggested to use such approach to save keys to redis uniquely so we dont need to flush all keys inside redis after test finished.

So in such case, the actual path to some keys inside redis will be:

/tmp/tmpmhsq9m28/keypair1_storage.pub

But without key specifying:

.acrakeys/keypair1_storage.pub